lib/teleterm/config.go

```diff
@@ -51,10 +51,6 @@ func (c *Config) CheckAndSetDefaults() error {
 		return trace.BadParameter("missing network address")
 	}
 
-	if c.PrehogAddr == "" {
-		return trace.BadParameter("missing prehog address")
-	}
```

Comment:
What happens in tshd when `PrehogAddr` is empty?

I made this changes and ran the debug app (with usage reporting enabled in app config). Then I ssh'd a couple of times through Connect but I didn't see logs about reporting being disabled or any failures when submitting events.

```diff
diff --git a/lib/teleterm/daemon/usagereporter.go b/lib/teleterm/daemon/usagereporter.go
index 2437b0d03d..0506c354e4 100644
--- a/lib/teleterm/daemon/usagereporter.go
+++ b/lib/teleterm/daemon/usagereporter.go
@@ -34,12 +34,12 @@ import (
 const (
 	// minBatchSize determines the size at which a batch is sent
 	// regardless of elapsed time
-	minBatchSize = 25
+	minBatchSize = 2
 
 	// maxBatchSize is the largest batch size that will be sent to
 	// the server; batches larger than this will be split into multiple
 	// requests.
-	maxBatchSize = 50
+	maxBatchSize = 3
 
 	// maxBatchAge is the maximum age a batch may reach before
 	// being flushed, regardless of the batch size
@@ -49,7 +49,7 @@ const (
 	// may grow. Events submitted once this limit is reached will be discarded.
 	// Events that were in the submission queue that fail to submit may also be
 	// discarded when requeued.
-	maxBufferSize = 100
+	maxBufferSize = 4
 
 	// submitDelay is a mandatory delay added to each batch submission
 	// to avoid spamming the prehog instance.
diff --git a/web/packages/teleterm/src/env.ts b/web/packages/teleterm/src/env.ts
index e07441f99c..07462a1713 100644
--- a/web/packages/teleterm/src/env.ts
+++ b/web/packages/teleterm/src/env.ts
@@ -8,7 +8,7 @@ let ENV: Env;
 // process.env.NODE_ENV is resolved by webpack at compile time
 if (process.env.NODE_ENV === 'development') {
   ENV = {
-    prehogAddress: 'https://reporting-staging.teleportinfra.dev',
+    prehogAddress: '',
     feedbackAddress:
       'https://kcwm2is93l.execute-api.us-west-2.amazonaws.com/prod',
   };
```

Reply:
You didn't see anything because we blocked sending events from electron here
```ts
if (!(ENV.prehogAddress && isCollectingUsageMetricsEnabled))
```
With this condition removed you'd see: 
```
failed to submit batch of usage events: unavailable: unsupported protocol scheme "", unavailable: unsupported protocol scheme "" batch_size:2 usagereporter/usagereporter.go:166
```